src/dbnode/encoding/options.go

```diff
@@ -61,6 +63,7 @@ func newOptions() Options {
 		byteFieldDictLRUSize:   defaultByteFieldDictLRUSize,
 		iStreamReaderSizeM3TSZ: defaultIStreamReaderSizeM3TSZ,
 		iStreamReaderSizeProto: defaultIStreamReaderSizeProto,
+		metrics:                NewMetrics(instrument.NewOptions().MetricsScope()),
```

Comment:
Not really sure if we should be creating new `instrument.Options` here, disconnected from the globally used one.

Reply:
This is done in other places, too. E.g. 

https://github.com/m3db/m3/blob/a2a70d0168c272415bb914a6aaab1e803b7d77cf/src/dbnode/storage/options.go#L208-L213

https://github.com/m3db/m3/blob/a2a70d0168c272415bb914a6aaab1e803b7d77cf/src/dbnode/storage/bootstrap/bootstrapper/fs/options.go#L81-L83

To me, it seems that the pattern is to initialize options with some sensible defaults and then inject dependencies using methods (which sometimes is not very convenient).